packages/client/tests/functional/interactive-transactions/prisma/_schema.ts

```diff
@@ -0,0 +1,38 @@
+import { idForProvider } from '../../_utils/idForProvider'
+
+export default ({ provider }) => {
+  const id = idForProvider(provider)
+  return /* Prisma */ `
+  generator client {
+    provider        = "prisma-client-js"
+    previewFeatures = ["interactiveTransactions"]
+  }
+  
+  datasource db {
+    provider = "${provider}"
+    url      = env("DATABASE_URI_${provider}")
+  }
+  
+
+  // / User model comment
+  model User {
+    id    ${id}
+    email String  @unique
+    // / name comment
```

Comment:
I think we can clean these comments every time you see them. These don't work in this form, anyways.